va-gov/pages/search.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Search Results
+layout: page-react.html
+entryname: search
+includeBreadcrumbs: true
+---
```

Comment:
I think we need a `production: false` and possibly `preview: false` here to keep it out of Vets.gov and VA.gov.